.github/workflows/update-openapi-ui.yml

```diff
@@ -0,0 +1,51 @@
+# Update the dependencies of the OpenAPI UI and rebuild it, then create a pull request with the changes
+
+name: Update OpenAPI UI dependencies
+
+on: workflow_dispatch
+
+jobs:
+  update-openapi-ui:
+    name: Update OpenAPI UI Dependencies
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./dev/com.ibm.ws.openapi.ui/swagger-ui
+    steps:
+    - name: Checkout Open Liberty
+      uses: actions/checkout@v2
+    - name: Use Node.js v16
+      uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+        cache: 'npm'
+        cache-dependency-path: ./dev/com.ibm.ws.openapi.ui/swagger-ui/package-lock.json
+    - name: Update OpenAPI UI dependencies
+      run: |
+        npx --yes npm-check-updates -u
+        npm update
+    - name: Find new Swagger UI version
+      run: |
+        export SWAGGER_UI_VERSION=$(jq -r '.packages."node_modules/swagger-ui".version' package-lock.json)
+        echo "SWAGGER_UI_VERSION=$SWAGGER_UI_VERSION" >> $GITHUB_ENV
+        echo "Swagger UI version: $SWAGGER_UI_VERSION"
+    - name: Copy SCSS from Swagger UI source
+      run: |
+        git clone --depth 1 -b v${SWAGGER_UI_VERSION} https://github.com/swagger-api/swagger-ui.git swagger-ui-src
+        rm src/style/original/*
+        cp swagger-ui-src/src/style/* src/style/original
+        rm -r swagger-ui-src
+    - name: Build OpenAPI UI
+      run: npm run build -- --mode=production
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v3
```

Comment:
who is peter evans?

Reply:
When I looked for an action to create a PR from a workflow, this was the most widely used one.

Reply:
I've updated it to use a hash rather than a tag so that we won't pull in an updated version automatically.